hypervisor/esxi/esxi.go

```diff
@@ -97,6 +86,7 @@ func (p *EsxiHypervisorProvider) Name() string {
 func init() {
 	hypervisor.RegisterProvider("esxi", &EsxiHypervisorProvider{})
 	viper.SetDefault("hypervisor.esxi-insecure", true)
+	viper.SetDefault("hypervisor.vCenter", false)
```

Comment:
This parameter breaks the existing naming convention. It would be better to start with ``esxi-`` prefix.

Reply:
vCenter doesn't have anything to do with esxi though? It's just the endpoint.

Maybe it makes more sense to make a separate optional config field in the .toml file?

```
[vcenter]
enabled = true/false
```